test/e2e/Apps/BasicDotNetIsolated/OrchestrationQuery.cs

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Net;
+using Grpc.Core;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Azure.Functions.Worker.Http;
+using Microsoft.DurableTask.Client;
+
+namespace Microsoft.Azure.Durable.Tests.E2E;
+public static class OrchestrationQueryFunctions
+{
+    [Function(nameof(GetAllStatus))]
+    public static async Task<HttpResponseData> GetAllStatus(
+        [HttpTrigger(AuthorizationLevel.Anonymous, "get", "post")] HttpRequestData req,
+        [DurableClient] DurableTaskClient client)
+    {
+        try 
+        {
+            var instances = client.GetAllInstancesAsync();
+
+            var response = req.CreateResponse(HttpStatusCode.OK);
+            await response.WriteAsJsonAsync(instances);
+            return response;
+        }
+        catch (RpcException ex) 
+        {
+            var response = req.CreateResponse(HttpStatusCode.BadRequest);
+            response.Headers.Add("Content-Type", "text/plain");
+            await response.WriteStringAsync(ex.Message);
+            return response;
+        }
+    }
+
+    [Function(nameof(GetRunningStatus))]
+    public static async Task<HttpResponseData> GetRunningStatus(
+        [HttpTrigger(AuthorizationLevel.Anonymous, "get", "post")] HttpRequestData req,
+        [DurableClient] DurableTaskClient client)
+    {
+        try 
+        {
+            OrchestrationQuery filter = new OrchestrationQuery(Statuses: new List<OrchestrationRuntimeStatus> { OrchestrationRuntimeStatus.Running, OrchestrationRuntimeStatus.Pending, OrchestrationRuntimeStatus.Suspended });
```

Comment:
nit: wrap this into multiple lines. It's recommended to keep line lengths less than 120 chars or so.

```suggestion
            OrchestrationQuery filter = new OrchestrationQuery(Statuses: new List<OrchestrationRuntimeStatus>
            {
                OrchestrationRuntimeStatus.Running,
                OrchestrationRuntimeStatus.Pending,
                OrchestrationRuntimeStatus.Suspended,
            });
```

Reply:
Done